reco_utils/dataset/wikidata.py

```diff
@@ -0,0 +1,102 @@
+    r = requests.get(url+name.replace(" ", "%20"))
+    entityID = list(r.json()["query"]["pages"].keys())[0]
+    try:
+        entityID = r.json()["query"]["pages"][entityID]["pageprops"]["wikibase_item"]
+    except:
+        entityID = "entityNotFound"
+    return entityID
+
+def query_entity_links(entityID):
+    """Query all linked pages from a wikidata entityID
+    Args:
+        entityID (str): A wikidata page ID.
+    Returns:
+        data (json): dictionary with linked pages.
+    """
+    query = """
+    PREFIX entity: <http://www.wikidata.org/entity/>
+    #partial results
+
+    SELECT ?propUrl ?propLabel ?valUrl ?valLabel
+    WHERE
+    {
+        hint:Query hint:optimizer 'None' .
+        {	BIND(entity:"""+entityID+""" AS ?valUrl) .
+            BIND("N/A" AS ?propUrl ) .
+            BIND("identity"@en AS ?propLabel ) .
+        }
+        UNION
+        {	entity:"""+entityID+""" ?propUrl ?valUrl .
+            ?property ?ref ?propUrl .
+            ?property rdf:type wikibase:Property .
+            ?property rdfs:label ?propLabel
+        }
+
+        ?valUrl rdfs:label ?valLabel
+        FILTER (LANG(?valLabel) = 'en') .
+        OPTIONAL{ ?valUrl wdt:P18 ?picture .}
+        FILTER (lang(?propLabel) = 'en' )
+    }
+    ORDER BY ?propUrl ?valUrl
+    LIMIT 500
+    """
+    url = 'https://query.wikidata.org/sparql'
+    r = requests.get(url, params = {'format': 'json', 'query': query})
+    data = r.json()
+    return data
+
+def read_linked_entities(data):
+    """Obtain lists of liken entities (IDs and names) from dictionary
+    Args:
+        data (json): dictionary with linked pages.
+    Returns:
+        related_entities (list): List of liked entityIDs
+        related_names (list): List of liked entity names
+    """
+    related_entities = []
+    related_names = []
+    for c in data["results"]["bindings"]:
+        related_entities.append(c["valUrl"]["value"].replace("http://www.wikidata.org/entity/", ""))
+        related_names.append(c["valLabel"]["value"])
+    return (related_entities, related_names)
+
+def query_entity_description(entityID):
+    """Query entity wikidata description from entityID
+    Args:
+        entityID (str): A wikidata page ID.
+    Returns:
+        description (str): Wikidata short description of the entityID
+                           'descriptionNotFound' will be returned if no 
+                           description is found
+    """
+    query = """
+    PREFIX wd: <http://www.wikidata.org/entity/>
+    PREFIX schema: <http://schema.org/>
+
+    SELECT ?o
+    WHERE 
+    {
+      wd:"""+entityID+""" schema:description ?o.
+      FILTER ( lang(?o) = "en" )
+    }
+    """
+    url = 'https://query.wikidata.org/sparql'
+    r = requests.get(url, params = {'format': 'json', 'query': query})
+    try:
+        description = r.json()["results"]["bindings"][0]["o"]["value"]
+    except:
+        description = "descriptionNotFound"
+    return description
```

Comment:
you can run [black](https://github.com/python/black) on this after you've finished changes, this will format the code to match the style used in the repo.